### DISCUSSION TITLE
Integration with Microsoft 365/Teams

### DISCUSSION THREAD

**Discussion by Lauren-NaturalPoint** [GitbookIO/community#116] [Feature Requests]
### What's your feature/idea?

We'd like the option to integrate Microsoft 365 with GitBook. Primarily an integration with Teams and embedding content from 365 similar to Google Sheets, Docs, etc.

### What's your desired outcome?

Similar to a Slack integration and Google apps. 

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by rhys-the-davies** [comment]
helloo, I just came across this issue after seeing Microsoft Office listed as an existing integration but not being able to find it in the app. @addisonschultz is this something that's in the plan? Or does it already exist as the website implies? 
![Screenshot from 2024-08-15 10-46-26](https://github.com/user-attachments/assets/b7cc86e3-cd01-4adf-99fa-db454272a2ad)

**Reply by addisonschultz** [reply]
Hi @rhys-the-davies @Lauren-NaturalPoint ! These integrations are already supported out of the box, without needing to install anything! 

Any of the integrations listed in this section can all be unfurled (displayed / embedded) after pasting the link directly in GitBook - you can already try it out!

**Reply by shlomof-pyle** [reply]
when i try to embed certain microsoft links, they dont embed right, the just show as a link box, like this:
![image](https://github.com/user-attachments/assets/d3ccba94-3d33-4800-8ed4-5262bb68c5d8)

**Reply by addisonschultz** [reply]
Hi @shlomof-pyle - private links or links that don't have public permissions, they will appear as link blocks as you see here. Can you double check that the links you're pasting don't have any private permissions set?

**Comment by shlomof-pyle** [comment]
This is the link, it embeds fine in this email for example
How Warning Flags Works.webm<https://lanzar.sharepoint.com/:v:/s/knowledge/EX4X7VFzFwxBu66o630_9kEBK2sn7NR4J1DpkcJvuwaAsw?e=ZRLUEn&referrer=Outlook.Desktop&referrerScenario=email-linkwithembed>

________________________________

**Reply by addisonschultz** [reply]
We use the https://iframely.com/ service to show rich embeds in the GitBook editor - certain links like the one you're sharing, don't seem to work unfortunately. Can you let me know what type of content you're trying to embed? Maybe there is another way to show the content you're working with!

**Comment by shlomof-pyle** [comment]
Its microsoft stream videos. Maybe i need to download them as regular videos, and just share them from sharepoint that way
________________________________

**Reply by zdmilot** [reply]
@shlomof-pyle If you upload videos to your gitbook space as files, you can use them to embed into your documentation. Once a video has been uploaded to your file repository, click download (this will bring you to a site/URL where you can view the video), copy this URL of the video download link and use it to create a new embedded URL block... Anndddd voila you have just posted a video into your gitbook without the need to share them via a url. 


Also you can, If using stream videos, copy the embed code from Microsoft stream. As GitBook natively supports HTML you can try copying and pasting this into the markdown file or you can find the URL listed within this embed code and use that to embed your URL!

**Comment by shlomof-pyle** [comment]
Thanks, it was a cookie issue where the login to microsoft stream wasnt working, i had cookies blocked in the browser
________________________________
